src/components/landingPage/upload.tsx

```diff
@@ -1,218 +1,123 @@
-import { useEffect, useState } from "react";
+import { useEffect, useRef, useState } from "react";

```

Comment:
Nit: Watch your imports, it looks like you aren't using `useRef` anywhere in the code.

Reply:
Your linter _should_ be yelling at you about this. If it's not, something isn't configured correctly.

Reply:
TY! No it's not yelling. Would be good to get some proper linting in place.